roles/kubernetes-apps/persistent_volumes/aws/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+bin_dir: /usr/local/bin
+kube_config_dir: "/etc/kubernetes"
+persistent_volumes_enabled: false
+storage_classes:
```

Comment:
We should add this in a commented out format in a new file inventory/group_vars/all/aws.yaml
We have similar logic for azure and openstack already. These files are very helpful for new users.

Reply:
This should be renamed to aws_storage_classes